### PR TITLE
JBPM-5954 - Process Instance Description Lines begin with spaces

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
@jomarko do you mind reviewing when you get the chance? Thanks.

**Comment by jomarko** [comment]
Before save
screenshot from 2017-05-10 08-01-44

After save:
screenshot from 2017-05-10 08-02-26

**Comment by tsurdilo** [comment]
@jomarko nice catch - thanks. i have retired the "Text" editor which did not maintain multi-lines. Should be good now. Fix amended.

File: jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/propertywindow.js [hunk 1]
```diff
@@ -6037,13 +6056,13 @@ Ext.form.ConditionExpressionEditorField = Ext.extend(Ext.form.TriggerField,  {
...
+        function initCodeEditor(myTypeMode, myShowLineNumbers, myDoLineWrapping, myDoMatchBrackets) {
             this.foldFunc = CodeMirror.newFoldFunction(CodeMirror.braceRangeFinder);
             sourceEditor = CodeMirror.fromTextArea(document.getElementById(scriptEditor.getId()), {
-                mode: "text/x-java",
-                lineNumbers: true,
-                lineWrapping: true,
-                matchBrackets: true,
+                mode: myTypeMode,
+                lineNumbers: myShowLineNumbers,
+				lineWrapping: myDoLineWrapping,
```
Comment (jomarko) [inline jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/propertywindow.js]: Please keep the indentation same in whole block.

File: jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/propertywindow.js [hunk 2]
```diff
@@ -6062,18 +6081,29 @@ Ext.form.ConditionExpressionEditorField = Ext.extend(Ext.form.TriggerField,  {
...
     }
 });
 
+
+Ext.form.ConditionExpressionEditorTextField = Ext.extend(Ext.form.ConditionExpressionEditorField,  {
+    typemode : "text",
+    showLineNumbers : false,
+    doLineWrapping : false,
+    doMatchBrackets : false,
+	editorTitle : ORYX.I18N.PropertyWindow.text,
```
Comment (jomarko) [inline jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/propertywindow.js]: Please keep the indentation in whole block.

**Comment by jomarko** [comment]
@tsurdilo it works fine now. Thanks. Please just incorporate my two comments above and I can approve.

File: jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/propertywindow.js (hunk 1 above)
Comment (hasys) [inline jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/propertywindow.js]: It looks like tabulation sign usage. @tsurdilo you can disable "Use tab character" and IDEA will replace Tabs by whitespaces automatically. To do it: Settings > Editor > Code Style > Java > Tabs and Indents > and uncheck "Use tab character"

**Comment by tsurdilo** [comment]
@jomarko amended - thanks!

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by manstis** [comment]
@hasys Are you OK with this now? @tsurdilo is desperate for it to be merged so he can do some more work

**Pull request merged**